### PR TITLE
Issue #9243 fixed.

### PR DESCRIPTION
**Pull request by suraj-webkul**
## Issue Reference

Issue #9243 fixed.

### CODE REVIEW

**Review by amit-webkul — APPROVED** [review]

File: packages/Webkul/Theme/src/Repositories/ThemeCustomizationRepository.php [hunk 1]
```diff
@@ -19,24 +19,54 @@ public function model(): string
...
+        if (in_array($data['type'], ['image_carousel', 'services_content'])) {
+            $this->uploadImage(request()->all(), $theme);
+        }
+
+        return $theme;
+    }
+
     /**
      * Upload images
      *
```
Comment (shivendra-webkul) [inline packages/Webkul/Theme/src/Repositories/ThemeCustomizationRepository.php:53]: Need to define type of the variable $theme

Comment (suraj-webkul) [reply packages/Webkul/Theme/src/Repositories/ThemeCustomizationRepository.php:53]: done

Comment (phyothiha) [reply packages/Webkul/Theme/src/Repositories/ThemeCustomizationRepository.php:54]: @suraj-webkul 

[In this code](https://github.com/bagisto/bagisto/blob/322e29c3cb1c986af5aa71c98f850cd3ca02632b/packages/Webkul/Theme/src/Repositories/ThemeCustomizationRepository.php#L94-L96), title key is missing. Without it, there is no title key in the database and blank in UI.

```
$options['images'][] = [
    'image' => 'storage/' . $path,
    'link'  => $image['link'],
    'title' => $image['title'], // <-- missing key
];
```

### Video

https://github.com/bagisto/bagisto/assets/12383757/ddce9b14-cb9a-4fab-a724-8c66aa34f119

**Comment by ashishkumar-webkul** [comment]
@suraj-webkul 

Check and resolve the Conflicts which are appearing in the PR.

**Comment by suraj-webkul** [comment]
Hello, @ashishkumar-webkul sir.
There is no conflicts.

File: packages/Webkul/Theme/src/Repositories/ThemeCustomizationRepository.php [hunk 2]
```diff
@@ -19,24 +19,57 @@ public function model(): string
...
+    public function update($data, $id): ThemeCustomization
+    {
+        $locale = core()->getRequestedLocaleCode();
+
+        if ($data['type'] == 'static_content') {
+            $data[$locale]['options']['html'] = preg_replace('/<script\b[^>]*>(.*?)<\/script>/is', '', $data[$locale]['options']['html']);
+            $data[$locale]['options']['css'] = preg_replace('/<script\b[^>]*>(.*?)<\/script>/is', '', $data[$locale]['options']['css']);
+        }
+
+        $data['status'] = request()->input('status') == 'on';
```
Comment (devansh-webkul) [inline packages/Webkul/Theme/src/Repositories/ThemeCustomizationRepository.php:37]: Request class cannot be used inside the repository.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Theme/src/Repositories/ThemeCustomizationRepository.php (hunk 2 above)
Comment (suraj-webkul) [reply packages/Webkul/Theme/src/Repositories/ThemeCustomizationRepository.php]: done

**Comment by ashishkumar-webkul** [comment]
@suraj-webkul 

Checked and Found that the Image crousal image adding and currrent slider delete is working fine. Please find the video for reference.

**Add Image in Image Crousal**


https://github.com/bagisto/bagisto/assets/92721837/90872227-3631-4b8f-bd13-05a5c6342cb0


**Delete Slider**


https://github.com/bagisto/bagisto/assets/92721837/00d94503-f5c0-40ea-b00b-3f9572fc465b

**Pull request closed without merging**